force-app/main/default/lwc/accordionSection/accordionSection.css

```diff
@@ -0,0 +1,12 @@
+@media print {
+    .slds-accordion__content {
+        overflow: visible !important;
```

Comment:
Can we move this into the recentSessionsOverrides and scope the css so we can remove the !important?